packages/projects-docs/pages/learn/getting-started/setting-up-ios.mdx

```diff
@@ -1,21 +1,46 @@
+Bring your CodeSandbox projects with you wherever you go with just your iPhone or iPad. Simply download the CodeSandbox app from the [App Store](https://apps.apple.com/us/app/codesandbox/id1423330822) and log in using your CodeSandbox account. If you don't have one yet, you can easily sign up using a GitHub, Google, or Apple account.
 
-<br/>
-## Importing a project
+For the full CodeSandbox experience, we recommend linking your GitHub account. This will give you access to all of our features and capabilities.
 
-Once you have signed into CodeSandbox with your GitHub account, you can import your first project. 
+### Select the Appropriate Workspace
 
-You can easily import any public repositories by pasting its GitHub URL into the “GitHub repository URL” field and hitting the “enter” key or the “Import button”. This action will automatically fork the repository and spin a development environment for you to start coding straight away.
+Upon logging in, your personal workspace will be selected as the default. If you're part of a team, you can easily switch to your team's workspace by selecting it from the sidebar. On an iPhone, you can access the screen to switch workspaces by tapping the **Back** button located in the top left corner of the screen.
 
-In addition, you can create a project from any repository (public or private if you are a Pro user) of any GitHub Organization you belong to or create a new project using one of the templates available.
+![Screenshot of the iOS Dashboard](/ios-dashboard.png)
 
-![IMG_01E76C8C3559-1.jpg](../images/IMG_01E76C8C3559-1.jpg)
\ No newline at end of file
+### Accessing Sandboxes and Repositories
+
+If you are an existing CodeSandbox user and after selecting a workspace, you will be presented with a list of sandboxes and repositories that you have recently interacted with. To view all available sandboxes and repositories, simply navigate through the different categories in the sidebar or use the **Search** functionality available at the top of the screen.
+
+### Creating a New Sandbox or Repository
+
+You also have the option of starting from scratch by creating a new sandbox or importing a repository from GitHub.
+
+#### Importing a Repository from GitHub
+
+To import a repository from GitHub, simply tap on the **New Repository** button to start the import process. If you haven't already, you'll be prompted to link your CodeSandbox account with a GitHub account.
+
+In the import screen, you can either enter the link to the GitHub repository you wish to import or select a repository from one of the GitHub organizations you belong to, including your personal GitHub organization. CodeSandbox will automatically create a fork in GitHub for repositories you do not have write permissions for.
```

Comment:
```suggestion
To import a repository from GitHub, click on the **New Repository** button to start the import process. You'll be prompted to link your CodeSandbox account with a GitHub account if you haven't already.

In the import screen, you can either enter the link to the GitHub repository you want to import or select a repository from one of the GitHub organizations you belong to, including your personal GitHub organization. CodeSandbox will automatically create a fork in GitHub for repositories for which you do not have write permissions.
```

Reply:
I will make some tweaks. On iOS apps the "tap" action is preferred over "click".